core/types/types.go

```diff
@@ -18,19 +18,18 @@ package types
 
 import (
 	"errors"
-	"math/big"
-
 	scale "github.com/ChainSafe/gossamer/codec"
 	"github.com/ChainSafe/gossamer/common"
+	"math/big"
```

Comment:
i think this will cause a goimports error, could you move this back under "errors"